undertone/undertone-htb-validator.js

```diff
@@ -35,6 +35,15 @@ function partnerValidator(configs) {
     var result = Inspector.validate({
         type: 'object',
         properties: {
+            schain: {
```

Comment:
Hi @tamarm-perion, from our testing I noticed your Schain object doesn't follow openRTB schema. Can you confirm if this is intended? If not, would you be able to create another PR with the changes? I've included the reference to the spec here: https://github.com/InteractiveAdvertisingBureau/openrtb/blob/master/supplychainobject.md

Reply:
Hi @lksharma,

Thank you for the review.
If possible, we would like to use the current implementation and parse supply-chain nodes in our server.

Reply:
As per the spec, the supply-chain nodes are expected to be an Array of SupplyChainNode objects. With the current implementation, you would only parse a single node. Can you confirm if this is the intended purpose?

Reply:
Hi @lksharma,

Thanks for the reply,
I made another PR according to your remarks: https://github.com/indexexchange/ix-library-adapters/pull/239/files

Reply:
@tamarm-perion,

Thank you for the quick changes. I added some additional comments to the new pull request.